### DISCUSSION TITLE
在公有云上部署本地连接不上

### DISCUSSION THREAD

**Discussion by Hank-LaiHQ** [apache/rocketmq#7621] [Q&A]
我现在是使用的单节点部署的4.9.4版本，其中broker部署在阿里云的服务器上，nameserver部署在华为云的服务器上，我尝试过在broker.conf文件和broker-a.properties文件上配置了brokerIP1 = broker公网ip 和 broperIP2 = broker公网ip，然后我在启动broker的时候我通过查看nameserver的日志，注册了一个新的broker，broker的ip地址显示的是阿里云的内网地址，new broker registered, 172.31.128.172:10911 HAServer: 172.31.128.172:10912，然后本地启动客户端的时候报了Caused by: org.apache.rocketmq.remoting.exception.RemotingConnectException: connect to 172.31.128.172:10911 failed这个错误，无法使用阿里云的内网ip地址连接上broker

**Comment by leizhiyuan** [comment]
说明你启动的可能不是这个配置文件，可以看看自己启动之后，通过./bin/mqadmin getBrokerConfig -b "xxx:10091" |grep brokerIP1

看看brokerip1是不是对的。
